lib/node_modules/@stdlib/blas/ext/base/sdssum/test/test.ndarray.native.js

```diff
@@ -45,7 +45,7 @@ tape( 'main export is a function', opts, function test( t ) {
 });
 
 tape( 'the function has an arity of 4', opts, function test( t ) {
-	t.strictEqual( sdssum.length, 4, 'has expected arity' );
+	t.strictEqual( sdssum.length, 4, 'returns expected value' );
```

Comment:
The changes in this file are incomplete. You should refactor to avoid the use of `floor`. See reference PRs linked to in the OP.